socorro/unittest/processor/test_breakpad_pipe_to_json.py

```diff
@@ -251,172 +248,156 @@ class TestCase(TestCase):
         info = ['OS', ]
         d = DotDict()
         bpj._extract_OS_info(info, d)
-        ok_('system_info' in d)
-        eq_(d.system_info, {})
+        assert 'system_info' in d
+        assert d.system_info == {}
 
     def test_extract_CPU_info(self):
         info = ['CPU', 'x86', 'GenuineIntel family 6 model 22 stepping 1', 1]
         d = DotDict()
         bpj._extract_CPU_info(info, d)
-        ok_('system_info' in d)
-        eq_(
-            d.system_info,
-            {
-                "cpu_arch": 'x86',
-                "cpu_info": 'GenuineIntel family 6 model 22 stepping 1',
-                "cpu_count": 1
-            }
-        )
+        assert 'system_info' in d
+        expected = {
+            "cpu_arch": 'x86',
+            "cpu_info": 'GenuineIntel family 6 model 22 stepping 1',
+            "cpu_count": 1
+        }
+        assert d.system_info == expected
 
     def test_extract_OS_and_CPU_info(self):
         info = ['OS', 'Windows NT', '5.1.2600 Service Pack 2']
         d = DotDict()
         bpj._extract_OS_info(info, d)
         info = ['CPU', 'x86', 'GenuineIntel family 6 model 22 stepping 1', 1]
         bpj._extract_CPU_info(info, d)
-        ok_('system_info' in d)
-        eq_(
-            d.system_info,
-            {
-                'os': 'Windows NT',
-                'os_ver': '5.1.2600 Service Pack 2',
-                "cpu_arch": 'x86',
-                "cpu_info": 'GenuineIntel family 6 model 22 stepping 1',
-                "cpu_count": 1
-            }
-        )
+        assert 'system_info' in d
+        expected = {
+            'os': 'Windows NT',
+            'os_ver': '5.1.2600 Service Pack 2',
+            "cpu_arch": 'x86',
+            "cpu_info": 'GenuineIntel family 6 model 22 stepping 1',
+            "cpu_count": 1
+        }
+        assert d.system_info == expected
 
     def test_extract_crash_info(self):
         info = ['Crash', 'EXCEPTION_ACCESS_VIOLATION_READ', '0x676c', 1]
         d = DotDict()
         crashing_thread = bpj._extract_crash_info(info, d)
-        ok_('crash_info' in d)
-        eq_(
-            d.crash_info,
-            {
-                "type": 'EXCEPTION_ACCESS_VIOLATION_READ',
-                "crash_address": '0x676c',
-                "crashing_thread": 1
-            }
-        )
-        eq_(crashing_thread, 1)
+        assert 'crash_info' in d
+        expected = {
+            "type": 'EXCEPTION_ACCESS_VIOLATION_READ',
+            "crash_address": '0x676c',
+            "crashing_thread": 1
+        }
+        assert d.crash_info == expected
+        assert crashing_thread == 1
 
     def test_extract_module_info(self):
         info = ['Module', 'firefox.exe', '24.0.0.4925', 'firefox.pdb',
                 '9FFDDF56AADE45988C759EF5ABAE53862', '0x00400000',
                 '0x004e0fff', '1']
         d = DotDict()
         bpj._extract_module_info(info, d, 17)
-        ok_('modules' in d)
-        ok_(len(d.modules), 1)
```

Comment:
This is one of a bunch of `ok_` that probably should have been `eq_`.

Reply:
rgr. I think I saw this above too